IngestionGrain/Ingestions.cs

```diff
@@ -1,15 +1,31 @@
 ﻿using CommunAxiom.Commons.Client.Contracts.Ingestion;
+using CommunAxiom.Commons.Ingestion;
 using Orleans;
-using System;
+using Orleans.Runtime;
 using System.Threading.Tasks;
 
-namespace IngestionGrain
+namespace CommunAxiom.Commons.Client.Grains.IngestionGrain
 {
     public class Ingestions : Grain, IIngestion
     {
-        public Task<string> TestGrain(string Grain)
+        private readonly Business _business;
```

Comment:
Since the business object is aware of an ongoing process, this grain should support reentrancy so you can monitor the ingestion while it's happening. https://dotnet.github.io/orleans/docs/grains/reentrancy.html